bitnami/phpmyadmin/Chart.yaml

```diff
@@ -29,4 +29,4 @@ name: phpmyadmin
 sources:
   - https://github.com/bitnami/bitnami-docker-phpmyadmin
   - https://www.phpmyadmin.net/
-version: 7.0.1
+version: 7.0.2
```

Comment:
Due to the amount of changes, I would say this is a "minor" change (and not "patch"), i.e. it should be changed to "7.1.0".

Reply:
I've changed it. Can you resolve conflict?